server/Makefile-lorax-punica

```diff
@@ -0,0 +1,12 @@
+lorax_punica_commit := c71861a653412267dc27ec86013dd945ce3474bc
```

Comment:
that would be preferred however I'm not sure lorax are built and available, their CI process does not seem to compile a prebuilt container/kernels. If there is a way to avoid this build step I'd love to 🙂

Reply:
Ah yeah, they have releases, so I kinda assumed that they upload wheels to PyPI.

Reply:
yea unfortunately it looks like only the client is published to pypi